github/repos.go

```diff
@@ -462,6 +462,40 @@ type ListContributorsOptions struct {
 	ListOptions
 }
 
+// EnableVulnerabilityAlerts Enables vulnerability alerts and the dependency graph for a repository.
```

Comment:
nit: s/Enables/enables/
so that the auto-generated Godocs read nicely.

Reply:
done